# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1510184784 18000
#      Wed Nov 08 18:46:24 2017 -0500
# Node ID 3d26a174e143a9ceceb62d57ffac7a8ffa94d14f
# Parent  2391bd89adf81a0839cb3192695662df305221ae
* variables.cc (extract_function): Omit 'first' from error message.

diff --git a/libinterp/corefcn/variables.cc b/libinterp/corefcn/variables.cc
--- a/libinterp/corefcn/variables.cc
+++ b/libinterp/corefcn/variables.cc
@@ -127,17 +127,17 @@ extract_function (const octave_value& ar
                   const std::string& trailer)
 {
   octave_function *retval = nullptr;
 
   retval = is_valid_function (arg, warn_for, 0);
 
   if (! retval)
     {
-      std::string s = arg.xstring_value ("%s: first argument must be a string",
+      std::string s = arg.xstring_value ("%s: argument must be a string",
                                          warn_for.c_str ());
 
       std::string cmd = header;
       cmd.append (s);
       cmd.append (trailer);
 
       int parse_status;
 
